types/react-native-calendars/index.d.ts

```diff
@@ -8,7 +8,11 @@
 
 import * as React from 'react';
 import { StyleProp, TextStyle, ViewStyle } from 'react-native';
-export import LocaleConfig = require('xdate');
+import XDateLocaleConfig = require('xdate');
```

Comment:
Can this be changed to

`import * as XDateLocaleConfig ...` to match the styles above?

Reply:
Hi @Tyler-Zhang 

When I replace `import XDateLocaleConfig = require('xdate');` with `import * as XDateLocaleConfig from 'xate';` I have an error raised on import:


> This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.ts(2497)